drivers/storage/portworx/manifest/manifest.go

```diff
@@ -312,6 +307,35 @@ func fillPrometheusDefaults(
 	}
 }
 
+func fillTelemetryDefaults(
+	rel *Version,
+) {
+	pxVersion, err := version.NewSemver(rel.PortworxVersion)
+	if err != nil || !pxutil.IsCCMGoSupported(pxVersion) {
+		// Old CCM Java telemetry
+		if rel.Components.Telemetry == "" {
+			rel.Components.Telemetry = defaultCCMJavaImage
+		}
+		if rel.Components.MetricsCollector == "" {
+			rel.Components.MetricsCollector = defaultCollectorImage
+		}
+		if rel.Components.MetricsCollectorProxy == "" {
```

Comment:
let's reset images of the new ccm to empty, just in case customer downgrade. add a unit test to upgrade then downgrade and add test case in testrail too

Reply:
We don't need to reset the images here. The `Version` object can send all the images from the manifest. We handle what needs to be set from this in the desired images in `portworx.go`.

Reply:
we don't support downgrade, but on safe side I'll reset those to empty. as when px is supported it should only run new ccm